contributors/design-proposals/volume-provisioning.md

```diff
@@ -412,9 +419,13 @@ type StorageClass struct {
   // Parameters for dynamic volume provisioner.
   Parameters map[string]string `json:"parameters,omitempty"`
 }
-
 ```
 
+StorageClass can have annotation
+`volume.beta.kubernetes.io/storage-class-selectors` that contains free-form
+english text with description of labels that are supported by the provisioner
+in pvc.spec.selector when provisioning a new PV for this StorageClass.
```

Comment:
I prefer a structured form, and I hope the admin has a tool to aid in authorship.